oc_build.sh

```diff
@@ -27,35 +27,35 @@ OC_PROCESS="oc -n ${PROJ_TOOLS} process -f ${PATH_BC} -p NAME=${NAME} -p SUFFIX=
 
 # Apply a template (apply or use --dry-run)
 #
-OC_APPLY="oc -n "${PROJ_TOOLS}" apply -f -"
+OC_APPLY="oc -n ${PROJ_TOOLS} apply -f -"
 [ "${APPLY}" ] || OC_APPLY+=" --dry-run"
 
+# Force to start a new build each time (apply or don't run)
+#
+OC_START_BUILD="oc -n ${PROJ_TOOLS} start-build ${NAME}-${SUFFIX} --follow=true"
+[ "${APPLY}" ] || OC_START_BUILD=""
```

Comment:
Please make sure that OC_APPLY and OC_START_BUILD aren't kicking off two builds.  We could try pointing OC_PROCESS to a file and consuming that with OC_START_BUILD.